Example/PaymentSheet Example/PaymentSheet Example/ExampleCustomDeferredCheckoutViewController.swift

```diff
@@ -123,13 +125,17 @@ class ExampleCustomDeferredCheckoutViewController: UIViewController {
             self.updateLabels()
 
             // Update PaymentSheet with the latest `intentConfig`
-            self.paymentSheetFlowController.update(intentConfiguration: self.intentConfig) { error  in
-                if error != nil {
-                    // Retry
-                    self.updateUI()
+            self.paymentSheetFlowController.update(intentConfiguration: self.intentConfig) { [weak self] error in
+                if let error = error {
+                    print(error)
+                    self?.displayAlert("\(error)")
+                    // Retry - product code should use an exponential backoff
```

Comment:
```suggestion
                    // Retry - production code should use an exponential backoff
```

Reply:
I was testing invalid intent configs and realized this retry was dangerous to recommend w/o a delay.  Otherwise, an invalid config might spam enough update requests to have the merchant rate limited by Stripe.

cc @tillh-stripe